src/cclib/parser/orcaparser.py

```diff
@@ -934,6 +935,229 @@ def energy_intensity(line):
                 polarizability.append(line.split())
             self.polarizabilities.append(numpy.array(polarizability))
 
+        if line.strip() == 'ORCA-CASSCF':
+            self.skip_lines(inputfile, ['d', 'b'])
+            # -------------------------------------------------------------------------------
+            #                               ORCA-CASSCF
+            # -------------------------------------------------------------------------------
+            #
+            # Symmetry handling      UseSym     ... ON
+            # Point group                       ... C2
+            # Used point group                  ... C2
+            # Number of irreps                  ... 2
+            #    Irrep    A has   10 SALCs (ofs=   0) #(closed)=   0 #(active)=   2
+            #    Irrep    B has   10 SALCs (ofs=  10) #(closed)=   0 #(active)=   2
+            #  Symmetries of active orbitals:
+            #    MO =    0  IRREP= 0 (A)
+            #    MO =    1  IRREP= 1 (B)
+            vals = next(inputfile).split()
+            if vals[0] == 'Symmetry':
+                # Symmetry section is only printed if symmetry is used
+                assert vals[-1] == 'ON'
+                point_group = next(inputfile).split()[-1]
+                used_point_group = next(inputfile).split()[-1]
+                num_irreps = int(next(inputfile).split()[-1])
+                num_active = 0
+                for i, line in zip(range(num_irreps), inputfile):
+                    reg = r'Irrep\s+(\w+) has\s+(\d+) SALCs \(ofs=\s*(\d+)\) #\(closed\)=\s*(\d+) #\(active\)=\s*(\d+)'
+                    groups = re.search(reg, line).groups()
+                    irrep = groups[0]
+                    salcs, ofs, closed, active = map(int, groups[1:])
+                    num_active += active
+                self.skip_line(inputfile, 'Symmetries')
+                for i, line in zip(range(num_active), inputfile):
+                    reg = r'(\d+)  IRREP= (\d+) \((\w+)\)'
+                    groups = re.search(reg, line).groups()
+                    mo, irrep_idx, irrep = groups
+
+            line = next(inputfile)
+            while line != 'SYSTEM-SPECIFIC SETTINGS:\n':
```

Comment:
Is this robust on Windows?

Reply:
Good question, probably not. Can anyone who uses Windows chime in?